test.sh

```diff
@@ -10,13 +13,16 @@ if [ ! -f "$SUBSTRATE_PATH" ]; then
     exit 2
 fi
 
-# Purge dev chain and then spin up the substrate node in background
+$SUBSTRATE_PATH --version
+
+echo "_____Purging dev chain_____"
 $SUBSTRATE_PATH purge-chain --dev -y
+echo "_____Spinning up the substrate node in background_____"
 $SUBSTRATE_PATH --dev &
 SUBSTRATE_PID=$!
 
-# Execute tests
+echo "_____Executing tests_____"
 yarn && yarn test
```

Comment:
We could do 
`yarn upgrade --pattern @polkadot` 
to always test with the latest Polkadot JS API version.

Reply:
Done, thanks for the suggestion!